packages/flutter_tools/test/commands.shard/permeable/build_apk_test.dart

```diff
@@ -117,65 +116,17 @@ void main() {
 
   group('Gradle', () {
     Directory tempDir;
-    ProcessManager mockProcessManager;
+    FakeProcessManager processManager;
     String gradlew;
     AndroidSdk mockAndroidSdk;
     TestUsage testUsage;
 
     setUp(() {
       testUsage = TestUsage();
-
       tempDir = globals.fs.systemTempDirectory.createTempSync('flutter_tools_packages_test.');
       gradlew = globals.fs.path.join(tempDir.path, 'flutter_project', 'android',
           globals.platform.isWindows ? 'gradlew.bat' : 'gradlew');
-
-      mockProcessManager = MockProcessManager();
```

Comment:
95% unused, but we don't get any warnings from un-used mocks so they probably just persisted years after they were relevant.